backend/main.py

```
@@ -1,369 +1,407 @@
+        if not result.get("success"): raise HTTPException(status_code=400, detail=result.get("message", "Failed to accept."))
+        return schemas.GenericResponse(success=True, message="Request accepted.", details=result.get("details"))
+    except HTTPException: raise
     except Exception as e:
-        logger.error(f"Error fetching updates via ContentAgent: {e}", exc_info=True)
-        raise HTTPException(status_code=500, detail=f"An internal error occurred while fetching updates: {str(e)}")
+        logger.error(f"Error in /accept: {e}", exc_info=True)
+        raise HTTPException(status_code=500, detail=str(e))
+
+@app.get("/api/v1/content/updates", response_model=List[schemas.Update], tags=["Content Agent"])
+async def get_real_time_updates():
+    try: return await content_agent.fetch_updates()
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)